conda-recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,12 @@
+numpy:
+    - '1.25'
+c_compiler:                   # [linux]
+    - gcc                     # [linux]
```

Comment:
Do we need to add `c_stdlib` and `c_stdlib_version` and to update `meta.yaml` to replace `sysroot_linux-64 >=2.28  # [linux]` with `{{ stdlib('c') }}`? Or would you prefer to handle that by a separate PR?

Reply:
This is a good idea in general and may as well go in this PR, good point